.evergreen.yml

```diff
@@ -277,13 +277,14 @@ functions:
       shell: bash
       script: |
         ${PREPARE_SHELL}
-        # Only record to cypress cloud if this is a pr or a mainline commit.
-        if [[ "${requester}" == "github_pr" || "${requester}" == "commit" || "$requester" == "patch" ]]; then
-          yarn cy:run --record --key "${parsley_cypress_record_key}" --reporter junit
+        # Allow spec filtering for an intentional patch.
+        if [[ "${requester}" == "patch" ]]; then
+          yarn cy:run --reporter junit --spec "${cypress_spec}"
```

Comment:
i think this already works without it, but should we add the `parameters` block like we have in spruce? I'm not really sure what benefit the block gives though 😅

Reply:
Sure just added it for consistency! 
I was kind of curious about what it did too since I didn't see it in the docs.
I think it does 2 things; applies default values for patch params that are loaded in as expansions and also provides a description when using the evergreen list command in the cli.